### ISSUE TITLE
[BUG] error in grabManifestJsonFromRepository

### ISSUE THREAD

**Issue by QuaCKeReD** [TfTHacker/obsidian42-brat#42]
Not sure when issue started, but now under Obsidian v0.16.0, the following errors are appearing;

```
error in grabManifestJsonFromRepository for https://raw.githubusercontent.com/Quorafind/Obsidian-Bionic-Reading/HEAD/manifest-beta.json Error: Request failed, status 404
    at new t (app.js:1:722395)
    at Nw (app.js:1:722587)
    at app.js:1:723264
    at app.js:1:235192
    at Object.next (app.js:1:235297)
    at a (app.js:1:234036)
```

This is same for any plugins added.

Impact seems to be the `-beta` in the manifest.json filename - all repos listed as failing have a `manifest.json`, but none have `manifest-beta.json`

**Comment by QuaCKeReD** [comment]
Maybe I have jumped gun - is there supposed to be a manifest-beta.json file?

Am wondering now if this is why some plugins added dont 'work' 😛

**Comment by TfTHacker** [comment]
no this is a bug, thanks for reporting it. It seems to be something new for sure. but I have a solution. Will be posting today a beta of BRAT, and hopefully posting to public soon.

if you update brat with brat :-), you should get the fix.

**Comment by QuaCKeReD** [comment]
How very meta 😀

**Comment by TfTHacker** [comment]
This is now resolved with 0.6.35 update to BRAT.
